source/2019-07-26-the-ember-times-issue-108.md

```diff
@@ -1,14 +1,16 @@
 ---
 title: The Ember Times - Issue No. 108
-author: the crowd
+author: Isaac Lee, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/07/26-the-ember-times-issue-108.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+This week:
+
+learn how to use Sentry to track production errors 📈,
 
```

Comment:
Would it be a good idea to change the title to include something about the migration from the add-on?

Reply:
Thanks for your feedback, Anne! I wouldn't ignore anyone's suggestions, no worries.

I agree with you that Tobias' blog post emphasizes what differences a user will encounter between the addon and the official package. I think the title here can remain as is, in case there are readers who have never tried out Sentry before.

Is that okay with you?

Reply:
True :) That makes sense too!